examples/ChimneySwap/app/src/main/java/xyz/thepathfinder/www/chimneyswap/Chimney.java

```diff
@@ -0,0 +1,74 @@
+package xyz.thepathfinder.www.chimneyswap;
+
+import android.graphics.Bitmap;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.google.gson.JsonObject;
+
+public class Chimney {
+
+    private int id;
+    private String name;
+    private String imageUrl;
+    private Bitmap image;
+    private LatLng position;
+
+    public Chimney() {
+        this.id = 0;
```

Comment:
They are not no-ops, it is explicit initialization. Even if Java will do the same thing without them, we should keep it. Not everyone knows Java's default initialization rules.